### PR TITLE
Add Stable Diffusion with KerasCV and OpenVINO notebook

### PR DESCRIPTION
**Pull request by as-suvorov**
Ticket: CVS-133140

### CODE REVIEW

File: notebooks/290-stable-diffusion-keras-cv/290-stable-diffusion-keras-cv.ipynb [hunk 1]
```diff
@@ -0,0 +1,722 @@
+{
```
Comment (eaidova) [inline notebooks/290-stable-diffusion-keras-cv/290-stable-diffusion-keras-cv.ipynb:1]: <p>can you please extend description a lottle bit more?</p><p>Maybe something like that:</p><p>Stable Diffusion is a powerful, open-source text-to-image generation model. There are multiple implementations of this pipeline in different frameworks. Previously, we already considered how to convert and optimize PyTorch Stable Diffusion using HuggingFace Diffusers library (link on notebook). In this tutorial, we consider how to convert and run Stable Diffusion from <a href="https://www.tensorflow.org/tutorials/generative/generate_images_with_stable_diffusion" target="_blank">KerasCV</a> that employs graph mode execution, which enhances performance by leveraging graph optimization and enabling parallelism and in the same time maintains a user-friendly interface for image generation. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (eaidova) [inline notebooks/290-stable-diffusion-keras-cv/290-stable-diffusion-keras-cv.ipynb:1]: <p>I believe workflow diagradm will be helpful</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (eaidova) [inline notebooks/290-stable-diffusion-keras-cv/290-stable-diffusion-keras-cv.ipynb:1]: <p>I belive it should be before any other descriptions, not inside conversion block. Usually we separate dependencies installation into prerequisites block (sometimes, users want to run notebooks several times like demo, but having it inside conversion block, makes feeling that all of them need only for conversion. Also I think note about python requirement also should be more highlighted, if it is not a part of conversion section</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (eaidova) [inline notebooks/290-stable-diffusion-keras-cv/290-stable-diffusion-keras-cv.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    %pip install keras-cv "keras==2.15" numpy "tensorflow==2.15.1" openvino-nightly gradio
</pre><p>please add -q flag to install command, there is also some issues with installing openvino-nightly on windows platform if user already have previously installed openvino package. For avoiding that we usually uninstall previously used openvino packages</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (eaidova) [inline notebooks/290-stable-diffusion-keras-cv/290-stable-diffusion-keras-cv.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #6.        "photograph of an astronaut riding a horse",
</pre><p>generated image quality not looks great.. also I do not see seed in this line, it means that user will have different result from run to run here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (eaidova) [inline notebooks/290-stable-diffusion-keras-cv/290-stable-diffusion-keras-cv.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #8.    demo = gr.Interface(
</pre><p>please disable flagging for interface</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (eaidova) [inline notebooks/290-stable-diffusion-keras-cv/290-stable-diffusion-keras-cv.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #12.            gr.Slider(0, 200, value=45, label="Seed"),
</pre><p>is it whole supported range for seed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (as-suvorov) [reply notebooks/290-stable-diffusion-keras-cv/290-stable-diffusion-keras-cv.ipynb:1]: <p>Prerequisites block added. Any ideas on additional python 3.9 note highlighting?</p>

Comment (as-suvorov) [reply notebooks/290-stable-diffusion-keras-cv/290-stable-diffusion-keras-cv.ipynb:1]: <p>Diagram added</p>

Comment (as-suvorov) [reply notebooks/290-stable-diffusion-keras-cv/290-stable-diffusion-keras-cv.ipynb:1]: <p>Seed added</p>

Comment (as-suvorov) [reply notebooks/290-stable-diffusion-keras-cv/290-stable-diffusion-keras-cv.ipynb:1]: <p>Flagging disabled</p>

Comment (as-suvorov) [reply notebooks/290-stable-diffusion-keras-cv/290-stable-diffusion-keras-cv.ipynb:1]: <p>Seed range set to <code>0, 10000000</code> , taken from another NB </p>

Comment (as-suvorov) [reply notebooks/290-stable-diffusion-keras-cv/290-stable-diffusion-keras-cv.ipynb:1]: Description applied, thank you!

Comment (as-suvorov) [reply notebooks/290-stable-diffusion-keras-cv/290-stable-diffusion-keras-cv.ipynb:1]: Fixed

Comment (eaidova) [reply notebooks/290-stable-diffusion-keras-cv/290-stable-diffusion-keras-cv.ipynb:1]: not sure that it works for all possible preview format, maybe we can make it using [!NOTE] markdown syntax:
https://github.com/sinsukehlab/NOTE-test/blob/main/NOTE.md
?

Comment (eaidova) [reply notebooks/290-stable-diffusion-keras-cv/290-stable-diffusion-keras-cv.ipynb:1]: another possible way is using html 
https://gist.github.com/DanielKotik/4b81480c479a57e0dd13ac4d153e4451
